app/models/concerns/dossier_prefillable_concern.rb

```diff
@@ -3,11 +3,12 @@
 module DossierPrefillableConcern
   extend ActiveSupport::Concern
 
-  def prefill!(champs_public_attributes)
-    return unless champs_public_attributes.any?
+  def prefill!(champs_public_attributes, identity_attributes)
+    return unless champs_public_attributes.any? || identity_attributes.any?
```

Comment:
```suggestion
    return if !champs_public_attributes.any? || !identity_attributes.any?
```
je crois que les conditions multiples avec un `unless` c'est pas trop notre truc (et a titre perso je lis ca plus facilement)